oxenmq/fmt.h

```diff
@@ -0,0 +1,29 @@
+#pragma once
+
+#include <fmt/format.h>
+#include "connections.h"
+#include "auth.h"
+#include "address.h"
+#include "oxenmq-internal.h"
```

Comment:
This needs to get dropped; this internal header is only for the local .cpp files, and doesn't get included in the `-dev` package headers.

The AuthLevel `to_string` in it can just get moved into `auth.h`